__tests__/app.test.js

```diff
@@ -0,0 +1,45 @@
+const data = require("../db/data/test-data/index.js");
+//console.log(categoryData);
+
+const seed = require("../db/seeds/seed");
+
+const request = require("supertest");
+
+const app = require("../app");
+
+const db = require("../db/connection.js");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => {
+  return db.end();
+});
+
+describe("/api/categories", () => {
+  test("status code 200, responds with an array of category objects", () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then(({ body }) => {
+        const { categories } = body;
+        expect(categories).toBeInstanceOf(Array);
```

Comment:
We are expecting the response to this end point to be an object with a key of topics, that has a value of an array of objects. If this does not make sense, have a read of your endpoints.json for an example.